tools/data-handler/src/utils/schemas.ts

```diff
@@ -11,24 +11,30 @@
 */
 
 import cardBaseSchema from '../../../schema/cardBaseSchema.json' with { type: 'json' };
-import cardsConfigSchema from '../../../schema/cardsConfigSchema.json' with { type: 'json' };
 import cardTypeSchema from '../../../schema/cardTypeSchema.json' with { type: 'json' };
+import cardsConfigSchema from '../../../schema/cardsConfigSchema.json' with { type: 'json' };
```

Comment:
Move this back; `cardsConfigSchema` is before `cardTypeSchema` in alphabetical order; similarly in `schemas` array.